versions/3.0.4.md

```diff
@@ -2682,7 +2684,7 @@ Other than the JSON Schema subset fields, the following fields MAY be used for f
 
 | Field Name                                      |                             Type                              | Description                                                                                                                                                                                                                                                                                                                                                                                                                                       |
 | ----------------------------------------------- | :-----------------------------------------------------------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| <a name="schema-nullable"></a>nullable           |                           `boolean`                           | A `true` value adds `"null"` to the allowed type specified by the `type` keyword, only if `type` is explicitly defined within the same Schema Object. Other Schema Object constraints retain their defined behavior, and therefore may disallow the use of `null` as a value. A `false` value leaves the specified or default `type` unmodified. The default value is `false`.                                                                    |
```

Comment:
The old wording seems to mean

> Mentally put the string value of `type` into an array and add the string `"null"` to this array, then interpret the array in the JSON Schema sense because OpenAPI explicitly forbids array values for the `type` keyword in the preceding section.

The new text hopefully avoids this mental gymnastics and the possible confusion of `null` as an allowed value and `"null"` as an additional string item in the array value of the JSON Schema `type` keyword.

Reply:
Not only are array values for type forbidden,  but `null` is also forbidden.
>  null is not supported as a type

But I'm not sure the new wording is better. In particular, the "allowed values" reference could be confused with `enum`. Oh, but actually we should add `null` to the enum if it exists. [JSON Schema](https://json-schema.org/draft-05/draft-wright-json-schema-validation-00.pdf) says:
> Elements in the array MAY be of any type, including null.
> An instance validates successfully against this keyword if its value is equal to one of the elements in this keyword's array value.

Oh my.

Reply:
I agree with @mikekistler - I find the new wording more confusing.  I would go with something like:

> This keyword only takes effect if `type` is explicitly defined within the same Schema Object.  A `true` value indicates that both `null` values and values of the type specified by `type` are allowed.  Other Schema Object constraints retain their defined behavior, and therefore may disallow the use of `null` as a value. A `false` value leaves the specified or default `type` unmodified. The default value is `false`.

Reply:
Took over Henry's proposal